### PR TITLE
feat: strf- 9673 Remove lodash from paper-handlebars deps

### PR DESCRIPTION
**Pull request by jairo-bc**
## What? Why?

Removed lodash from runtime dependencies (it's still used in dev dependencies)

## How was it tested?

npm test

----

cc @bigcommerce/storefront-team

### CODE REVIEW

File: helpers/3p/utils/lib/deepMatches.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    var len = array.length;
+    var i = -1;
+  
+    while (++i < len) {
+      if (deepMatches(array[i], value)) {
+        return true;
+      }
+    }
+    return false;
+  }
```
Comment (rafa-avila-bc) [inline helpers/3p/utils/lib/deepMatches.js:28]: Is this `}` misformatted? 🤔

File: helpers/contains.js [hunk 2]
```diff
@@ -12,7 +13,11 @@ const _ = require('lodash');
 const factory = () => {
     return function(container, value) {
         const options = arguments[arguments.length - 1];
-        const contained = _.includes(container, value);
+        if ( utils.isObject(container)) {
+            console.log(value, container, utils.contains(value, container));
```
Comment (rafa-avila-bc) [inline helpers/contains.js:17]: is this `log` debug-only? is so, can we delete it?

File: helpers/lib/fonts.js [hunk 3]
```diff
@@ -24,30 +24,32 @@ const fontProviders = {
...
-                if (!_.isArray(familyHash[familyKey])) {
+                if (!utils.isArray(familyHash[familyKey])) {
                     familyHash[familyKey] = [];
                 }
 
                 weights = weights.split(',');
 
                 familyHash[familyKey].push(weights);
-                familyHash[familyKey] = _.uniq(_.flatten(familyHash[familyKey]));
+                familyHash[familyKey] = [...new Set(familyHash[familyKey].flat())]
```
Comment (rafa-avila-bc) [inline helpers/lib/fonts.js:48]: nice! 👍🏾

**Review by rafa-avila-bc — APPROVED** [review]

**Review by jmwiese — APPROVED** [review]

File: helpers/3p/utils/lib/deepMatches.js (hunk 1 above)
Comment (jairo-bc) [reply helpers/3p/utils/lib/deepMatches.js:28]: yeah, I copied from previous file, where were 2 spaces instead of 4

**Review by rafa-avila-bc — APPROVED** [review]

**Pull request merged**